infra/modules/notifications/resources/email.tf

```
@@ -0,0 +1,5 @@
+resource "aws_pinpoint_email_channel" "app" {
+  application_id = aws_pinpoint_app.app.application_id
+  from_address   = var.sender_email != null ? (var.sender_display_name != null ? "${var.sender_display_name} <${var.sender_email}>" : var.sender_email) : null
```

if i understand correctly sender_email is a required variable, so can we simplify this to a single conditional like:

```suggestion
  from_address   = var.sender_display_name != null ? "${var.sender_display_name} <${var.sender_email}>" : var.sender_email)
```